dev/ci/scripts/ci-mczify.sh

```diff
@@ -7,9 +7,16 @@ ci_dir="$(dirname "$0")"
 
 git_download mczify
 
+git_download algebra_tactics
```

Comment:
please don't put multiple projects in 1 script

Reply:
Sorry, I missed the fact that it wouldn't help with https://github.com/coq/coq/pull/19925
Done, CI "green"

Reply:
#19925 doesn't care, it's just harder to understand when once script handles multiple projects